server/routes/visitors.test.ts

```diff
@@ -2,29 +2,20 @@ import type { Express } from 'express'
 import request from 'supertest'
 import { SessionData } from 'express-session'
 import * as cheerio from 'cheerio'
-import { Restriction, VisitorListItem, VisitSessionData } from '../@types/bapv'
+import { Restriction, VisitorListItem, VisitSessionData, VisitSlotList } from '../@types/bapv'
 import PrisonerVisitorsService from '../services/prisonerVisitorsService'
+import VisitSessionsService from '../services/visitSessionsService'
 import { appWithAllRoutes, flashProvider } from './testutils/appSetup'
 
+jest.mock('../services/prisonerVisitorsService')
+jest.mock('../services/visitSessionsService')
+
 let sessionApp: Express
-let prisonerVisitorsService: PrisonerVisitorsService
 const systemToken = async (user: string): Promise<string> => `${user}-token-1`
 
 let flashData: Record<'errors' | 'formValues', Record<string, string | string[]>[]>
-let returnData: { prisonerName: string; visitorList: VisitorListItem[] }
 let visitSessionData: VisitSessionData
 
-class MockPrisonerVisitorsService extends PrisonerVisitorsService {
```

Comment:
Changed the way services mocked as adding a second service caused lint errors about multiple class definitions in one file...